specification/saas/resource-manager/readme.csharp.md

```diff
@@ -10,6 +10,6 @@ csharp:
   payload-flattening-threshold: 1
   clear-output-folder: true
   client-side-validation: false
-  namespace: microsoft.saas
-  output-folder: $(csharp-sdks-folder)/saas/management/Microsoft.SaaS/GeneratedProtocol
+  namespace: saas
+  output-folder: $(csharp-sdks-folder)/services/preview/$(namespace)/mgmt/2018-03-01-beta/$(namespace)
```

Comment:
I believe this is not the right namespace and output-folder format for C#.

@allenjzhang could you please have a look?

Reply:
@allenjzhang @ArcturusZhang can this (right C# dir and ARM approcal) be expedited for today?
Tomorrow starts a lock-down on changes to PROD and I want to push the new manifest as soon as possible.
This is pending for two weeks now because @zhenglaizhang did not respond this whole time.
Thanks